pingcap/tidb
# metrics: add server status count

### What problem does this PR solve?


Issue Number: close #48291

Problem Summary:

### What is changed and how it works?

### Check List

Tests 

1. start a cluster
2. import tidb.json
3. check tidb->Server->TiDB server status on Grafana
![image](https://github.com/pingcap/tidb/assets/4352397/b07867c2-a6a8-400d-82fd-204eff4ec411)

  > - [ ] I checked and no code files have been changed.
  >   

Side effects


Documentation


### Release note



Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```
